test/timeline/test_timeline.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+// This test loads an individual raidboss timeline and makes sure
+// that timeline.js can parse it without errors.
+
+let fs = require('fs');
+let Timeline = require('../../ui/raidboss/timeline.js');
+
+let exitCode = 0;
+
+let inputFilename = String(process.argv.slice(2));
```

Comment:
This doesn't bother me so much? It's the same thing you'd have in a Python script called from elsewhere.  It's a little weird, but this just feels like any other command line intermediary between scripts in two languages.